src/overlay/template.jsx

```diff
@@ -193,7 +194,7 @@ export function Overlay({
               )}
               {!fullScreen && (
                 <div class="paypal-checkout-modal">
-                  <div class="paypal-checkout-logo">
+                  <div class="paypal-checkout-logo" dir="ltr">
```

Comment:
the logo should always be left to right:
![Screenshot 2024-01-24 at 6 09 00 PM](https://github.com/paypal/paypal-common-components/assets/3824954/e5c2e42e-de1c-4d9f-95ae-d0a38e1268ac)